lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java

```diff
@@ -4470,11 +4488,51 @@ private void createEditTitleDialog(UIContainer tofill, SimplePage page, SimplePa
 			UISelect.make(form, "cssDropdown", options, labels, "#{simplePageBean.dropDown}", currentCss);
 			
 			UIOutput.make(form, "cssDefaultInstructions", messageLocator.getMessage("simplepage.css-default-instructions"));
+			UIOutput.make(form, "ownerDefaultInstructions", messageLocator.getMessage("simplepage.owner-default-instructions")
+					.replace("{1}", messageLocator.getMessage("simplepage.permissions")).replace("{2}", messageLocator.getMessage("simplepage.more-tools")));
 			UIOutput.make(form, "cssUploadLabel", messageLocator.getMessage("simplepage.css-upload-label"));
 			UIOutput.make(form, "cssUpload");
 			UIBoundBoolean.make(form, "nodownloads", 
 					    "#{simplePageBean.nodownloads}", 
 					    (simplePageBean.getCurrentSite().getProperties().getProperty("lessonbuilder-nodownloadlinks") != null));
+
+			//Set the changeOwner dropdown in the settings dialog
+			UIOutput.make(form, "changeOwnerSection");
+			List<String> roleOptions = new ArrayList<String>();
+			List<String> roleLabels = new ArrayList<String>();
+				List<String> possOwners = new LinkedList<String>();
```

Comment:
I'm not sure what you mean for this one.  Do you want the empty line removed?  Everything else is consistently tab-indented.